AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponents_DepthOfFieldAdded.py

```diff
@@ -175,29 +326,404 @@ def AtomEditorComponents_DepthOfField_AddedToEntity():
 
-        # 14. Delete DepthOfField entity.
+        # 14. CameraEntityId Override can be turned on and off.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('CameraEntityId Override'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.cameraentityid_override_on,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('CameraEntityId Override')) is True)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('CameraEntityId Override'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.cameraentityid_override_off,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('CameraEntityId Override')) is False)
+
+        # 15. Enabled Override can be turned on and off.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enabled Override'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enabled_override_on,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enabled Override')) is True)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('Enabled Override'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.enabled_override_off,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('Enabled Override')) is False)
+
+        # 16. QualityLevel Override can be turned on and off.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('QualityLevel Override'), True)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.qualitylevel_override_on,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('QualityLevel Override')) is True)
+
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('QualityLevel Override'), False)
+        general.idle_wait_frames(1)
+        Report.result(
+            Tests.qualitylevel_override_off,
+            depth_of_field_component.get_component_property_value(
+                AtomComponentProperties.depth_of_field('QualityLevel Override')) is False)
+
+        # 17. ApertureF Override can be set to 0 and back to default value of 1.
+        depth_of_field_component.set_component_property_value(
+            AtomComponentProperties.depth_of_field('ApertureF Override'), 0)
```

Comment:
if this is a float set it to 0.0 and check for 0.0

Reply:
fixed